### ISSUE TITLE
[BUG] Extension settings are not accessed correctly

### ISSUE THREAD

**Issue by beatrycze-volk** [kitodo/kitodo-presentation#1300]
## Description

The extension configurations are accessed in the FlexForm settings array.

Examples:
`$this->settings['forceAbsoluteUrl']`
`$this->settings['fileGrpScore']`

## Reproduction

Steps to reproduce the behaviour:

1. Install TYPO3 11.x with PHP 8.x
2. Click on link to display document page e.g. [https://sachsendigital.ddev.site](https://sachsendigital.ddev.site/werkansicht?tx_dlf%5Bdouble%5D=0&tx_dlf%5Bid%5D=39&tx_dlf%5Bpage%5D=1&tx_dlf%5Bpagegrid%5D=0)
3. See error

## Expected Behavior

The extension configurations should be accessed as the extension configuration, so for the examples above it should be:
`$this->extConf['general']['forceAbsoluteUrl']`
`$this->extConf['files']['fileGrpScore']`

## Screenshots and Examples

<img width="880" alt="settingserror" src="https://github.com/user-attachments/assets/7d655d2e-5f80-491e-ac11-72ccb8ba0924">

## Environment

 - RDBMS version: MariaDB 10.3
 - TYPO3 version: 11.5.37
 - PHP version: 8.2

**Comment by sebastian-meyer** [comment]
This is a duplicate of #1325, isn't it?

**Comment by fschoelzel** [comment]
@beatrycze-volk 
Is this related to this fix https://github.com/kitodo/kitodo-presentation/issues/1358 ?

**Comment by beatrycze-volk** [comment]
> [@beatrycze-volk](https://github.com/beatrycze-volk) Is this related to this fix [#1358](https://github.com/kitodo/kitodo-presentation/pull/1358) ?

Yeah, it looks so. I would assume that in fact the problem was trying to access the extension config by the plugin FlexForm settings. Bug can be marked as resolved by https://github.com/kitodo/kitodo-presentation/pull/1358.

**Comment by sebastian-meyer** [comment]
Thank you!

Fixed by #1358.
